scripts/copyUninstrumentedCode.js

```diff
@@ -0,0 +1,55 @@
+// We exclude libraries and other content from our instrumentation approach.  This script copies that content in
```

Comment:
It would be helpfule to elaborate this code comment? Especially terms like `instrumentation approach` and `uninstrumented form`. State that the code copied to the target directory is required for testem to include at running tests.